tests/iio_stresstest.c

```diff
@@ -388,16 +398,28 @@ static void *client_thread(void *data)
 	return (void *)EXIT_FAILURE;
 }
 
-static unsigned long int clamp(const char *name, unsigned long int val,
+static unsigned long int sanitize_clamp(const char *name, const char *argv,
 		unsigned long int min, unsigned long int max)
 {
+
+	unsigned long int val;
+	char buf[20];
+
+	if (!argv) {
+		val = 0;
+	} else {
+		/* sanitized buffer by taking first 20 (or less) char */
+		iio_snprintf(buf, sizeof(buf), "%s",argv);
```

Comment:
nitpick: `iio_snprintf(buf, sizeof(buf), "%s",__space__argv);`

Reply:
fixed - sorry - need to get the formatter installed...